doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
Actually... the validator operator doesn't need list their own public key in the config (validators.txt). rippled knows to trust the key from the `[validator_token]`.
We currently [instruct](https://github.com/ripple/rippled/blob/develop/doc/manifest-tool-guide.md#validator-keys) operators to list the master public key, however that unnecessary step isn't included when using the new validator-keys tool.
All that to say: if a validator key is compromised, the operator will modify the config to add both the `[validator_key_revocation]` for the old keys and the `[validator_token]` for the new keys.